From 207bc1198e3a23ca12af51638781bfd0b50fae6f Mon Sep 17 00:00:00 2001
From: Ryan Lortie <desrt@desrt.ca>
Date: Wed, 11 Feb 2015 17:37:49 -0500
Subject: contacts-types: avoid ambiguous 'ListStore'

Now that GLib also has a ListStore, we need to explicitly state
Gtk.ListStore here to avoid ambiguity.
---
 src/contacts-types.vala | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/contacts-types.vala b/src/contacts-types.vala
index 84521f8..ec0ba82 100644
--- a/src/contacts-types.vala
+++ b/src/contacts-types.vala
@@ -47,7 +47,7 @@ public class Contacts.TypeSet : Object  {
   // Map from display name to TreeIter for all custom types
   private HashTable<string, TreeIter?> custom_hash;
 
-  public ListStore store;
+  public Gtk.ListStore store;
   private TreeIter other_iter;
   private TreeIter custom_iter;
 
@@ -56,11 +56,11 @@ public class Contacts.TypeSet : Object  {
     vcard_lookup_hash = new HashTable<unowned string, GLib.List<InitData*> > (str_hash, str_equal);
     custom_hash = new HashTable<unowned string, TreeIter? > (str_hash, str_equal);
 
-    store = new ListStore (2,
-			   // Display name or null for separator
-			   typeof(string?),
-			   // Data for standard types, null for custom
-			   typeof (Data));
+    store = new Gtk.ListStore (2,
+                               // Display name or null for separator
+                               typeof(string?),
+                               // Data for standard types, null for custom
+                               typeof (Data));
   }
 
   private void add_data_to_store (Data data, bool is_custom) {
-- 
cgit v0.12

